R/build-reference.R

```diff
@@ -350,7 +350,8 @@ data_reference_topic <- function(topic,
   # Sections that contain arbitrary text and need cross-referencing
   out$description <- as_data(tags$tag_description[[1]])
   out$opengraph <- list(description = strip_html_tags(out$description$contents))
-  out$usage <- as_data(tags$tag_usage[[1]])
+  out$usage <- list(title = tr_("Usage"),
```

Comment:
Can you please put on one line or use [tidyverse style](https://style.tidyverse.org/syntax.html#long-lines) here?

Reply:
Done, I used `styler` for reformatting it